imageroot/systemd/user/mattermost-app.service

```diff
@@ -21,6 +23,7 @@ ExecStart=/usr/bin/podman run --conmon-pidfile %t/mattermost-app.pid \
     --env MM_SQLSETTINGS_DRIVERNAME=postgres \
     --env MM_SQLSETTINGS_DATASOURCE=postgres://mattuser:Nethesis,1234@127.0.0.1:5432/mattermost?sslmode=disable&connect_timeout=10 \
     --env TZ=UTC \
+    --env=MM_* \
```

Comment:
```suggestion
    --env-file=%S/state/smarthost.env
```

Reply:
the file could be missing, I remove it if the smarthost is not enabled

Reply:
yes, that could be a problem
you must be sure the file exists, even if empty